nservicebus/hosting/docker-host/index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Docker Container Host
+summary: Take advantage of process isolation by hosting endpoints in Docker containers
+tags:
+ - Hosting
+related:
+ - nservicebus/dotnet-templates
+ - nservicebus/licensing
+ - samples/hosting/docker
+reviewed: 2018-02-28
+---
+
+Docker containers provide the ability to deploy endpoints in a self contained manner. To create an endpoint to host in a Docker container, download and install the [Docker endpoint template](/nservicebus/dotnet-templates.md), and create a new project using `dotnet new nsbdockerendpt`. The project that is created will have the required endpoint setup infrastructure in addition to the needed `DOCKERFILE` to create and deploy a container hosting one endpoint.
+
+## Licensing
+### Compliance
+See the [Licensing page](https://particular.net/licensing) for license specifics.
+
+When running Docker containers, the host operating system is considered a node for the purpose of licensing. If there are multiple containers running on the same machine, and those containers host NServiceBus endpoints in each of them, only a one node license is required for the host operating sytem.
```

Comment:
Is this something we want to have here? Do we have any other docs that go into this kind of licensing guidance?

Reply:
The other place that I thought it might belong was on the `nservicebus/licensing` page, but that doesn't talk at all about license compliance/types. The particular.net/licensing page doesn't talk about this kind of distinction either. We've had a number of questions about this that I've been privy to so I think it needs to go somewhere. 

The question is if this is the best place for it right now. Would a user look here, or find their way here, if they were asking a question about Docker licensing?